falco-no-driver.yaml

```
@@ -0,0 +1,77 @@
+package:
+  name: falco-no-driver
+  version: 0.36.2
+  epoch: 0
+  description: Cloud Native Runtime Security
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - bash
+      - bpftool
+      - build-base
+      - busybox
+      - c-ares-dev
+      - clang
+      - cmake
+      - curl-dev
+      - elfutils-dev
+      - gcc-12-default
+      - git
+      - grpc-dev
+      - libbpf
+      - libbpf-dev
+      - libelf-static
+      - libtool
+      - openssl-dev
+      - pkgconf
+      - protobuf-dev
+      - yaml-cpp-dev
+      - zlib-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/falcosecurity/falco
+      tag: ${{package.version}}
+      expected-commit: 1b62b5ccd1c64cd972ef0252262075cbf42a130c
+      recurse-submodules: true
+
+  - runs: |
+      mkdir skeleton-build && cd skeleton-build
+      cmake -DUSE_BUNDLED_DEPS=ON -DBUILD_FALCO_MODERN_BPF=ON -DCREATE_TEST_TARGETS=Off -DFALCO_VERSION=${{package.version}} ..
+      make ProbeSkeleton -j$(nproc)
```

Same here btw, should drop the -j and leave it to the build system:

```suggestion
      make ProbeSkeleton
```